### ISSUE TITLE
Authorization error code -600 on Android 8+

### ISSUE THREAD

**Issue by TristanMoreau** [okta/okta-oidc-js#647]
## I'm submitting this issue for the package(s):

- [ ] jwt-verifier
- [ ] okta-angular
- [ ] oidc-middleware
- [ ] okta-react
- [x] okta-react-native
- [ ] okta-vue

## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior

The EventEmitter emits a onError event "Authorization error" code -600 at the library load. This issue seems to manifest only on Android 8+ on physical devices. I managed to make it work with the AVD manager by simply restarting the emulator.
With the exact same codebase, everything is working as intended on one of my physical devices on Android 7 (I didn't try older versions).

## Expected behavior

It should load properly on Android 8+ or eventually throw a more specific error (maybe with an error code we can find on the okta documentation).

## Environment

- Package Version: 1.2.1
- Browser:
- OS: Android 8+
- Node version (`node -v`): 10.17.0
- Other:

**Comment by Krofi** [comment]
I got this exact same issue on all my devices from the sdk 19 to the sdk 24 included. All work fine from the sdk 25 to the latest.

In the documentation it is written that the minSdkVersion must be set to 19. Does this library is realy compatible? 

I will deploy my app in prod soon and I need it to be compatible with the sdk 21 minimum, could you reopen this case and help me to find a solution please?

**Comment by markrickert** [comment]
I spent lots a lot of hours on this exact problem :'( 

Everything installed, everything configured properly, works on ios, not on android.
Downloaded the sample react native project and input my config. works on ios, not on android. 

I stumbled upon this comment: https://devforum.okta.com/t/authorization-error-error-code-600-in-okta-react-native/6761/6

So I deleted the emulator instance in my AVD manager and recreated it with the same exact settings. Ran `yarn android` and 🤞 . This time login brought up chrome and chrome asked me a bunch of questions before ultimately spitting me out to the okta login screen. SUCCESS! No code or configuration changes at all. 

I'm not sure if it's because my emulator was a few months old or because it was created via the command line or what the problem was, but deleting and recreating the emulator seems to have solved my issue with the `Authorization error code -600` that was happening. For me it wasn't happening on config but when i called the `signIn()` function.

Hope this helps someone else not waste as much time on this as i did 🤦
